### PR TITLE
[TT-3602] Update swagger.yml for Policy CRUD APIs

### PR DESCRIPTION
**Pull request by jeffy-mathew**
Update swagger.yml for Policy CRUD APIs
## Description

Add policy CRUD API documentation in swagger.yml. Addition made through [PR](https://github.com/TykTechnologies/tyk/pull/3302)

### CODE REVIEW

File: user/policy.go [hunk 1]
```diff
@@ -5,6 +5,8 @@ import "gopkg.in/mgo.v2/bson"
 type GraphAccessDefinition struct {
 }
 
+// Policy represents a user policy
+// swagger:model
```
Comment (vaske) [inline user/policy.go:9]: 👏

**Review by vaske — APPROVED** [review]

**Comment by flyte** [comment]
Isn't it amazing when you suddenly hit a roadblock with your architectural design and then discover that it's already been fixed? :grin: 

What kind of time frame are we looking at for this feature (now that the functionality is merged and the documentation is on its way) to end up in a Tyk Gateway release please?

**Review by lghiur — APPROVED** [review]

**Comment by lghiur** [comment]
@flyte thanks and happy to hear this unblocks you, this will go out with our next patch release v4.0.1.

**Pull request merged**